### PR TITLE
Add two vBSDcon 2015 and two vBSDcon 2017 talks

### PR DESCRIPTION
**Pull request by gkwhite**

### CODE REVIEW

File: content/2015/vbsdcon/freebsd-Supporting_a_BSD_Project.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+layout: video
+title: "Supporting a BSD Project"
+date: 2015-09-13
+author: FreeBSD Foundation
```
Comment (lwhsu) [inline content/2015/vbsdcon/freebsd-Supporting_a_BSD_Project.md]: The author should be Ed Maste, emaste@

File: content/2017/vbsdcon/williams-Using_pkgsrc_for_multiplatform_deployments_in_heterogeneous_environments.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+layout: slides
+title: "Using pkgsrc for multi-platform deployments in heterogeneous environments, Made to Measure: Network Performance Analysis in FreeBSD"
```
Comment (lwhsu) [inline content/2017/vbsdcon/williams-Using_pkgsrc_for_multiplatform_deployments_in_heterogeneous_environments.md]: I think this title is combing titles from two talks. Are you sure about this editing?

And this talk (including the attached pdf file seems a pkgsrc talk, do you want to add this talk to papers.freebsd.org?

File: content/2015/vbsdcon/freebsd-Supporting_a_BSD_Project.md (hunk 1 above)
Comment (gkwhite) [reply content/2015/vbsdcon/freebsd-Supporting_a_BSD_Project.md]: fixed ... thanks!

File: content/2017/vbsdcon/williams-Using_pkgsrc_for_multiplatform_deployments_in_heterogeneous_environments.md (hunk 2 above)
Comment (gkwhite) [reply content/2017/vbsdcon/williams-Using_pkgsrc_for_multiplatform_deployments_in_heterogeneous_environments.md]: My bad.  Talk removed.   Thanks!

**Pull request merged**